integration-tests/test/common.py

```diff
@@ -23,6 +25,10 @@ class KeyPair:
     public_key: str
 
 
+    def sign_block_hash(self, block_hash: bytes) -> bytes:
```

Comment:
Shouldn't this be part of `pyrchain`?  I realize 2 PRs to separate repos are more problematic than 1 but if that's the only reason to not put this code in `pyrchain`, let's move `rchain/pyrchain` repo into `rchain/rchain` (eventually).

Reply:
There's currently no code related to p2p stuff in pyrchain unfortunately. Eventually, it'd be great if later, the generic code from this PR made it into pyrchain.

Reply:
I was suggesting @zsluedem move this code into that library or at least create a ticket for that.

Reply:
I was thinking about adapting the `PrivateKey` and `PublickKey` class in the pyrchain https://github.com/rchain/pyrchain/blob/master/rchain/crypto.py. That's a little bit out of the topic in this pr.

But you are totally right that I should create the ticket~
Here come the tickets.
https://rchain.atlassian.net/browse/RCHAIN-3642
https://rchain.atlassian.net/browse/RCHAIN-3643